packages/components/organisms/f-cookie-banner/gulpfile.js

```diff
@@ -9,26 +9,27 @@ const replace = require('gulp-replace');
 const LOCALES = ['en-GB', 'es-ES', 'it-IT', 'en-IE', 'en-AU', 'en-NZ', 'da-DK'];
 
 const PATHS = {
-    vueSrcFolder: './src',
-    tempDistFolder: './.tmp-dist',
-    tempVueFolder: './.tmp-vue',
-    rootDistFolder: '../f-cookie-banner/dist/static'
+    vueSrcFolder: './cookie-banner-static/src',
+    tempDistFolder: './cookie-banner-static/.tmp-dist',
+    tempVueFolder: './cookie-banner-static/.tmp-vue',
+    rootDistFolder: './dist/static',
```

Comment:
Nit: Is it worth making the naming more consistent, i.e., rename the `cookie-banner-static` folder simply to `static`?

Reply:
yeah, can do!